### ISSUE TITLE
Pretrained model?

### ISSUE THREAD

**Issue by cemoody** [sydsim/personlab-tf#2]
Hey! I took another spin of the system and I've been having issues getting it to train. Rather than going through those -- do you have a pretrained model that you're willing to upload? Thank you!

**Comment by sydsim** [comment]
Hi @cemoody !
I fixed some bugs recently, which were made up when I wrap up the code to share. I guess it will work now. I'm truly sorry for the inconvenience to you (and all people who struggled with my buggy code).
If you have still problem on training the model, I hope you to share with details (IMPORTANT!!) so we can improve this project.
Also, I can send you trained model backboned with MobilenetV2. Please leave your email address or send me an email if you want.

**Comment by cemoody** [comment]
Thank you so much @sydsim! I'm chrisemoody@gmail.com. So excited to try out the model!

**Comment by noumanriazkhan** [comment]
The pretrained is here https://www.dropbox.com/s/8vzpolhvh1cexj6/model.ckpt-86162.zip

**Comment by cemoody** [comment]
Thank you @noumanriazkhan! I'm a pytorch person myself so learning tensorflow is going slowly. Do you happen to have an example script or notebook hanging around that loads up the model in that zip and runs it on an image?

**Comment by noumanriazkhan** [comment]
Actually, we are on the same boat. I also want to run inference on single image but not so well versed with TF graphs. I am working on this will hopefully bring something up.

**Comment by noumanriazkhan** [comment]
@cemoody I have written the script for real time or single image inference and sent a pull request. Just wait for when it's accepted.

**Comment by intvo** [comment]
> @cemoody I have written the script for real time or single image inference and sent a pull request. Just wait for when it's accepted.

I tried to run your code and I got the following error 

<img width="1034" alt="screen shot 2018-11-11 at 5 48 44 pm" src="https://user-images.githubusercontent.com/40445787/48319292-130c3b00-e5da-11e8-8b8d-5245cc7118a5.png">

**Comment by noumanriazkhan** [comment]
Hi @intvo please set the batch_size=1 in config file.

**Comment by sydsim** [comment]
https://drive.google.com/file/d/1v_zVLxQSXI69jFIzOZoL0pRWQ1wwoz0f/view?usp=sharing
this is pretrained model based on MobileNetV2.

**Comment by nitba** [comment]
Hi @cemoody  and @noumanriazkhan  Have you measure the accuracy of model on coco dataset?

Thanks,
